doc/source/conf.py

```diff
@@ -46,8 +46,9 @@
               'ipython_sphinxext.ipython_console_highlighting',
               'sphinx.ext.intersphinx',
               'sphinx.ext.coverage',
-              'sphinx.ext.pngmath',
-              'sphinx.ext.ifconfig',
+              #'sphinx.ext.pngmath',
```

Comment:
I did see ``pngmath`` was deprecated. Is this needed for anything else though? can you build locally and give a check? (you can just ping here if ok)